criu/files-reg.c

```diff
@@ -1817,11 +1817,17 @@ static int do_open_reg(int ns_root_fd, struct reg_file_info *rfi, void *arg)
 	if (fd < 0)
 		return fd;
 
-	if ((rfi->rfe->pos != -1ULL) &&
+	/*
+	 * O_PATH opened files carry empty fops in kernel,
+	 * just ignore positioning at all.
+	 */
+	if (!(rfi->rfe->flags & O_PATH)) {
+		if ((rfi->rfe->pos != -1ULL) &&
 			lseek(fd, rfi->rfe->pos, SEEK_SET) < 0) {
```

Comment:
```
		if (rfi->rfe->pos != -1ULL &&
￼		    lseek(fd, rfi->rfe->pos, SEEK_SET) < 0) {
```

Reply:
Did you mean to just fix the indentation of `lseek`?

Reply:
Yes and you can remove extra brackets.